system/Router/RouteCollection.php

```diff
@@ -484,8 +484,10 @@ public function shouldAutoRoute(): bool
 
     /**
      * Returns the raw array of available routes.
+     *
+     * @param bool $only Whether to return without '*' routes.
```

Comment:
It took me a minute to figure this out. Maybe `$explicit` would be clearer? Or even `$excludeWildcard`?

Reply:
`explicit` is difficult to me. I changed to `$includeWildcard`.